### PR TITLE
Update internal-data-pipeline arch diagram and links

### PR DESCRIPTION
**Pull request by joselsegura**

### CODE REVIEW

File: docs/internal-pipeline/archive_sync_service_messages.md [hunk 1]
```diff
@@ -6,23 +6,23 @@ layout: default
 # Messages produced by archive-sync-service
 
 Archive Sync Service synchronizes every new archive by reading the related
-information from `ccx-XXX-insights-operator-archive-new` (where `XXX` needs to
-be replaced by environment, for example `prod`) Kafka topic, downloading the
-archive from AWS S3 and uploading it to DataHub (Ceph) bucket.
+information from `XXX-io-archive-new` (where `XXX` needs to be replaced by
```
Comment (epapbak) [inline docs/internal-pipeline/archive_sync_service_messages.md:9]: information from `[prod|qa]-io-archive-new`?

File: docs/internal-pipeline/archive_sync_service_messages.md [hunk 2]
```diff
@@ -6,23 +6,23 @@ layout: default
...
+environment, for example `prod`) Kafka topic, downloading the archive from
+AWS S3 and uploading it to DataHub (Ceph) bucket.
 
 Information about synchronized archive and its metadata are sent to
-`ccx-XXX-insights-operator-archive-synced` Kafka topic.
+`XXX-archive-synced` Kafka topic.
```
Comment (epapbak) [inline docs/internal-pipeline/archive_sync_service_messages.md:14]: same as above: [prod|qa]

File: docs/internal-pipeline/archive_sync_service_messages.md (hunk 1 above)
Comment (joselsegura) [reply docs/internal-pipeline/archive_sync_service_messages.md:9]: I was keeping the same way of explaining it that was already in place. Do you want to substitute all the XXX by [qa|prod] regexp?

File: docs/internal-pipeline/archive_sync_service_messages.md (hunk 2 above)
Comment (epapbak) [reply docs/internal-pipeline/archive_sync_service_messages.md:14]: Just replace all the XX with [qa|prod] please 😄

File: docs/internal-pipeline/rules_results_output.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+        ]
+```
+
+### Format of `pass` attribute in `Report` node
+
+This attribute contains list of rules that passes all conditions (i.e. rules without any problem/issue detected)
+
+### Format of `info` attribute in `Report` node
+
+TBD
```
Comment (epapbak) [inline docs/internal-pipeline/rules_results_output.md:191]: Are these TBDs on purpose, or did you maybe forget to update this part of the docs?

Comment (joselsegura) [reply docs/internal-pipeline/rules_results_output.md:191]: It was a copy/paste from previous file, tbh

**Pull request merged**